checkov/common/util/http_utils.py

```diff
@@ -204,36 +206,54 @@ def request_wrapper(
 
 
 async def aiohttp_client_session_wrapper(
-        url: str,
+        method: str,
+        url: StrOrURL,
         headers: dict[str, Any],
-        payload: dict[str, Any]
-) -> int:
+        payload: dict[str, Any] | None = None,
+) -> tuple[int, Any]:
     request_max_tries = int(os.getenv('REQUEST_MAX_TRIES', 3))
     sleep_between_request_tries = float(os.getenv('SLEEP_BETWEEN_REQUEST_TRIES', 1))
 
+    # 1. Read proxy URL (may include user:pass for authentication)
+    proxy_url = os.environ.get("HTTPS_PROXY", os.environ.get("HTTP_PROXY"))
+    proxy_auth = None
+    if proxy_url:
+        parsed_proxy_url = urlparse(proxy_url)
+        if parsed_proxy_url.username and parsed_proxy_url.password:
+            proxy_auth = aiohttp.BasicAuth(login=parsed_proxy_url.username, password=parsed_proxy_url.password)
+    # 2. Read path to custom certificate bundle
+    ca_bundle_path = os.getenv('BC_CA_BUNDLE')
```

Comment:
all env vars shoul be under `checkov/common/util/env_vars_config.py`, please use the values from there (also for http/s proxy above)